packages/common-utils/src/core/renderChartConfig.ts

```diff
@@ -190,10 +192,19 @@ const fastifySQL = ({
         case 'column_ref': {
           // FIXME: handle 'Value' type?
           const _n = node as ColumnRef;
-          // @ts-ignore
+          // @ts-expect-error _n exists
           if (typeof _n.column !== 'string') {
-            // @ts-ignore
-            colExpr = `${_n.column?.expr.value}['${_n.array_index?.[0]?.index.value}']`;
+            // @ts-expect-error assume column exists
```

Comment:
I know this method is only used for materialized fields. should we make sure they are Map type in this case? Just want to make sure this won't break for other types